scipy/stats/_qmc.py

```diff
@@ -1091,15 +1091,6 @@ class LatinHypercube(QMCEngine):
     ----------
     d : int
         Dimension of the parameter space.
-    centered : bool, optional
```

Comment:
`_random_cd` still references the `centered` keyword argument.

Reply:
Good catch thanks Matt